squad/frontend/views.py

```diff
@@ -482,6 +482,24 @@ def thresholds(request, group_slug, project_slug):
     return render(request, 'squad/thresholds.jinja2', context)
 
 
+@auth
+def toggle_outlier_metric(request, group_slug, project_slug, metric_id):
```

Comment:
I would use REST API but that can be changed later. I guess the bigger issue is that everyone can toggle the outlier flag. We probably need to address the permissions sooner than later.

Reply:
I was thinking of adding auth_write decorator here. And wrt UI, do you think that we shouldn't show the 'toggle outlier' option at all if the user hasn't got the write permissions for the corresponding project? Alternative is to show the error when he tries to perform the operation but I like the first solution better.